ApiContract/buyer/ad-details/request.json5

```diff
@@ -0,0 +1,7 @@
+//GET /api/products/{productId}
```

Comment:
In seller contract URL is /api/advertisement
Both should use either advertisement or product